c2corg_ui/static/partials/map/map.html

```diff
@@ -13,7 +13,12 @@
     </div>
   </div>
 
-  <button type="button" ng-show="mapCtrl.edit" ng-click="mapCtrl.resetFeature()"
-          class="btn btn-primary btn-xs map-reset-feature" translate>Reset feature</button>
+  <button type="button" ng-show="mapCtrl.canReset()" ng-click="mapCtrl.resetFeature()"
+          class="btn btn-primary btn-xs map-reset-feature"
+          title="{{ 'Reset geometry' | translate }}" translate>Reset geometry</button>
```

Comment:
> I think everybody knows at least the word geometry.

Everybody knows the word feature :P 
Nervermind, OK for me to change as you suggest.

> Yes, you can also delete point geometries.

I have just tested it. It works. Of course deleting the geometry for a WP prevent to save the document changes until a new geometry is created, so it's not that relevant to be able to delete a WP geometry. But at least it is consistent with other document types and we have point doc types (images!) where we should be able to remove the geometry.